cm_generate_makefile.bash

```diff
@@ -304,7 +304,7 @@ fi
 
 if [ ! -e ${KOKKOS_PATH}/CMakeLists.txt ]; then
    if [ "${KOKKOS_PATH}" == "" ]; then
-      CM_SCRIPT=`realpath $0`
+      CM_SCRIPT=`echo $0`
```

Comment:
Not sure what was wrong with `realpath` in the first place, but if you are not going to use it then just assign `$0` to `CM_SCRIPT`.